.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+# Inspired by https://discuss.circleci.com/t/how-to-test-multiple-versions-by-triggering-jobs-with-a-shell-function/11305/15.
+version: 2
+jobs:
+  build-golatest:
+    docker:
+      - image: circleci/golang
+    working_directory: /go/src/github.com/segmentio/analytics-go
+    steps:
+      - checkout
+      - run: make get vet test
```

Comment:
What do you think about adding a `ci` rule to the Makefile? Then you just do `make ci`, if it changes there's no need to update the circleci config.

Reply:
Done!